contents/docs/product-analytics/_snippets/identified-vs-anonymous-intro.mdx

```diff
@@ -0,0 +1,21 @@
+PostHog captures two types of events: [**anonymous** and **identified**](/docs/data/anonymous-vs-identified-events)
```

Comment:
Yeah. This is the annoying part about using snippet. It makes sense to include it in all the places except this one, but I also want to avoid duplicating just because of the hyperlink

Reply:
Could you duplicate just hyperlink part and use the rest of the snippet? It's just weird having a broken link.